pkg/stores/user/db.go

```diff
@@ -317,6 +322,7 @@ func (s *DBStore) toDBObj(u *entity.User) (*User, error) {
 		IsAdmin:     u.IsAdmin,
 		IsDirector:  u.IsDirector,
 		IsMod:       u.IsMod,
+		Notoriety:   u.Notoriety,
```

Comment:
with this, it may be impossible to .Set a user's notoriety back to zero.

Reply:
Why might it be impossible?

Reply:
zero values trap.

Reply:
oh, that dumb thing again, wouldn't this also apply to unmodding and unadminning people?

Reply:
[yes. `.Set` cannot be used for that purpose.](https://github.com/domino14/liwords/pull/535) 😅

Reply:
Added SetNotoriety to avoid this trap.